universal-gateway/examples/n8n.mdx

```diff
@@ -9,10 +9,9 @@ import CloudEndpoint from "/snippets/universal-gateway/_cloud-endpoint.mdx";
 import TryOut from "/snippets/universal-gateway/_try-out.mdx";
 
 
-
 n8n allows you to self-host, which is great when you're testing workflows, but when you're ready to turn your instance into a production-ready automation solution and don't want to buy a domain or pay for hosting, you might feel lost.
 
-ngrok works with the [self-hosted and enterprise editions](https://docs.n8n.io/hosting/) of n8n, including both Docker and npm-based installs, to securely expose them to the public internet.
+ngrok works with the [self-hosted and enterprise editions](https://docs.n8n.io/hosting/) of n8n, including Docker, Docker Compose, and npm-based installs, to securely expose them to the public internet.
```

Comment:
```suggestion
ngrok works with the [self-hosted and enterprise editions](https://docs.n8n.io/hosting/) of n8n, including Docker, Docker Compose, and npm-based installs, to make them securely available on the public internet.
```

I've gotten pushback from folks on using the word "expose" in these instances. I personally think it's literally the correct English word for what's happening here, but what're ya gonna do